### DISCUSSION TITLE
Add optional "Reference Title" field alongside all Title fields

### DISCUSSION THREAD

**Discussion by jsunsawyer** [craftcms/cms#8659] [Ideas]
The idea is that content editors often understand a single language. Linking up entries and categories in another language (site), when titles are translated, can be difficult.

The field would have options similar to the current Title field (Show the Title field? option, dynamic settings via twig, etc.)

The Title field would populate most instances of titles in the CP's UI. Anything that a user would find easier if it was in their language. Translated versions of titles still being editable where necessary.

This could also be tied in user accounts, adding an option for users to define which site's title they wish to see in the CP. This would require an optional field for each site. Just like path settings, though there may be a better way to handle that.

I feel like the addition of this, along with translate methods for titles https://github.com/craftcms/cms/issues/2064 and some better multi-site slug handling https://github.com/craftcms/cms/issues/849, managing multi-sites would feel so much better.

**Comment by brandonkelly** [comment]
You can pull this off by renaming “Title” to “Reference Title” from your entry type settings, and giving your entries a custom “Title” field with a handle of `realTitle`.

Your templates could then output:

```twig
<h1>{{ entry.realTitle ?? entry.title }}</h1>
```

**Comment by jsunsawyer** [comment]
@brandonkelly Unfortunately, this would change the title. We want to avoid changing the title so everything that uses titles can do so natively. Fields like LinkIt will use a localized version of a title when an entry or category is selected.

**Comment by jsunsawyer** [comment]
Reviewing some of my past request and gave this a bit more thought.

It would probably make more sense to allow editors to choose which language titles are shown in from the list of entries within sections. Then, when they click into an entry to edit it, they can see the translated title.

This could be a user option named something like `Show entry titles in my primary language` or similar – and each user could define their preferred language.

Obviously this requires a bit more thought, as sites are not always language based, but hope this gets the general idea across.
